core/services/synchronization/telemetry_ingress_batch_client.go

```diff
@@ -172,6 +215,7 @@ func (tc *telemetryIngressBatchClient) Send(ctx context.Context, telemData []byt
 	select {
 	case worker.chTelemetry <- payload:
 		worker.dropMessageCount.Store(0)
+		TelemetryClientMessagesSent.WithLabelValues(tc.url.String(), string(payload.TelemType)).Inc()
```

Comment:
at this point telemetry hasn't been sent successfully, it's just been passed to the worker that will send the telemetry

this metric should probably be moved into the worker [around here](https://github.com/smartcontractkit/chainlink/blob/develop/core/services/synchronization/telemetry_ingress_batch_worker.go#L71-L72) for a more accurate count. the `len(telemBatchReq.Telemetry)` would provide the number of telem messages sent